cmd/loom/loom.go

```diff
@@ -730,6 +733,17 @@ func loadApp(
 		return nil, err
 	}
 
+	if build := appStore.Get(minimumBuildKey); !bytes.Equal(build, []byte{}) {
+		minimumBuild := binary.BigEndian.Uint64(build)
+		currentBuild, err := strconv.ParseUint(loomchain.Build, 10, 64)
+		if err != nil {
+			currentBuild = 0
+		}
+		if currentBuild < minimumBuild {
+			return nil, fmt.Errorf("current build not supported (%d), require (%d)", currentBuild, minimumBuild)
+		}
+	}
+
```

Comment:
Also lets add a setting to `loom.yml` to skip this code, we may need to use custom builds sometimes:
```yaml
SkipMinBuildCheck: true
```